packages/dashboard/tsconfig.json

```diff
@@ -18,6 +18,7 @@
     "isolatedModules": true,
     "noEmit": true,
     "jsx": "react",
+    "noFallthroughCasesInSwitch": true
```

Comment:
From some googling, this is added to workaround a bug in react-scripts with typescript >= 4.0. According to this https://github.com/facebook/create-react-app/blob/master/CHANGELOG.md it has been fixed?

Reply:
Yes, it's fixed but it's added automatically to tsconfig.json every time we run `start:mock`.
![image](https://user-images.githubusercontent.com/11761240/101667032-9bfe3680-3a2d-11eb-840f-fb3ca4fe2a17.png)

Maybe we can issue a ticket to their project to solve that.

Reply:
I guess if we were to create a new typescript cra the option will be in tsconfig by default.